src/commands/accounts/list.ts

```diff
@@ -114,7 +115,7 @@ export const accountsList = async (
         tokenBalance,
         url: `${config.etherscanUrl}/address/${a.getId()}`,
         nftTokenUrl: `${config.etherscanUrl}/token/${nftTokenAddress}`,
-        nftBalance: withInventory ? await nft.balanceOf(a) : 0,
+        nftBalance: withInventory ? await nft721Api.balanceOf(a) : 0,
```

Comment:
I think `balanceOf` returns a BigNumber
```suggestion
        nftBalance: withInventory ? await nft721Api.balanceOf(a) : BigNumber.from(0),
```